frontend/range.ts

```diff
@@ -0,0 +1,11 @@
+function validRange (range: any) {
+    return range.min <= range.middle && range.middle <= range.max;
+}
+
+const range = {
+    min: 5,
+    middle: 10,    // TSLint will warn about unsorted keys here
```

Comment:
__[tslint]__
* The key 'middle' is not sorted alphabetically

<!-- Comment by [SideCI](https://sideci.com) -->